cmd/infra/aws/destroy.go

```diff
@@ -49,9 +50,10 @@ func NewDestroyCommand() *cobra.Command {
 	}
 
 	opts := DestroyInfraOptions{
-		Region: "us-east-1",
-		Name:   "example",
-		Log:    log.Log,
+		Region:              "us-east-1",
+		Name:                "example",
+		Log:                 log.Log,
+		AwsInfraGracePeriod: 30 * time.Minute,
```

Comment:
why are changing default? can we please keep current behaviour and let this be opt-int?

Reply:
addressed with [commit](https://github.com/openshift/hypershift/pull/2967/commits/4a34b10409ac327d80bb1cc62c28e2f224bcac1f)